.github/workflows/ci.yml

```
@@ -0,0 +1,56 @@
+name: CI
+
+on:
+  schedule:
+    - cron: "0 0 * * *"
+  push:
+    branches:
+      - 'main'
+    tags: ["v[0-9]+.[0-9]+[0-9]+"]
```

I guess this is a typo and you actually want to use 3 component versions.

```suggestion
    tags: ["v[0-9]+.[0-9]+.[0-9]+"]
```